mars/tensor/stats/tests/test_stats_execution.py

```diff
@@ -139,29 +141,25 @@ def test_t_test_execution(setup):
     if parse_version(scipy.__version__) >= parse_version("1.6.0"):
         alternatives = ["less", "greater", "two-sided"]
 
-        mt_from_stats = (
-            lambda a, b, alternative=None, equal_var=True: ttest_ind_from_stats(
-                a.mean(),
-                a.std(),
-                a.shape[0],
-                b.mean(),
-                b.std(),
-                b.shape[0],
-                alternative=alternative,
-                equal_var=equal_var,
-            )
+        mt_from_stats = lambda a, b, alternative=None, equal_var=True: ttest_ind_from_stats(
+            a.mean(),
+            a.std(),
+            a.shape[0],
+            b.mean(),
+            b.std(),
+            b.shape[0],
+            alternative=alternative,
+            equal_var=equal_var,
         )
-        sp_from_stats = (
-            lambda a, b, alternative=None, equal_var=True: sp_ttest_ind_from_stats(
-                a.mean(),
-                a.std(),
-                a.shape[0],
-                b.mean(),
-                b.std(),
-                b.shape[0],
-                alternative=alternative,
-                equal_var=equal_var,
-            )
+        sp_from_stats = lambda a, b, alternative=None, equal_var=True: sp_ttest_ind_from_stats(
+            a.mean(),
+            a.std(),
+            a.shape[0],
+            b.mean(),
+            b.std(),
+            b.shape[0],
+            alternative=alternative,
+            equal_var=equal_var,
```

Comment:
There is some failure of code style. Could you format this file with `black`?

Reply:
The following changes were made when I had run ``black`` in the first place. Should I ignore the changes made by ``black`` and manually restore them back to the original ones?

Reply:
You should upgrade `black` to the latest version.